can/interfaces/slcan.py

```diff
@@ -60,6 +60,8 @@ def __init__(self, channel, ttyBaudrate=115200, bitrate=None,
             baudrate of underlying serial or usb device
         :param int bitrate:
             Bitrate in bit/s
+        :param str btr:
+            BTR register value to set custom can speed
```

Comment:
Add a note about the relationship to `bitrate` (does it override?)

Reply:
Done